src/functional/iterator/embedded.py

```diff
@@ -66,8 +71,9 @@ def __init__(
 OffsetProvider: TypeAlias = dict[Tag, OffsetProviderElem]
```

Comment:
I think you proposed it earlier, but I ignored, because in iterator IR language int and tag are both offsets, not part of an offset, but I agree that for new people it's easier to thank in the way you propose. Will change.